tailormap-components/projects/core/src/lib/workflow/workflows/Workflow.helpers.ts

```diff
@@ -0,0 +1,48 @@
+import { GeoJSONGeometry } from 'wellknown';
+import { Geometry } from '../../shared/generated';
+import { Coordinate } from '../../user-interface/models';
+
+export class WorkflowHelpers {
+
+  public static findTopRight(geojson: GeoJSONGeometry | Geometry): Coordinate {
+    if (this.isGeneratedGeometry(geojson)) {
+      return this.findUpperRightCoordinate(geojson.coordinates[0] as number[][]);
```

Comment:
In general when accessing static methods the classname + method is used. I think both work but its more clear a static method is used instead of an instance method, so `WorkflowHelpers.findUpperRightCoordinate`